src/simplify_reshapes.cpp

```diff
@@ -627,6 +627,28 @@ struct find_transpose_contiguous_reshaper_unary
     }
 };
 
+struct find_broadcast_transpose
+{
+    auto matcher() const
+    {
+        return match::name("transpose")(match::arg(0)(match::name("multibroadcast").bind("bcast_ins")));
```

Comment:
multibroadcast instruction should only have once output. so need to add `used_once()`. 
Matcher is replacing that broadcast instruction with other.  Add test for that case as well.

Reply:
added

Reply:
Oh i misunderstood. I thought i was replacing multibroadcast. It is replacing transpose. So no need to use `used_once()`